util/tabComplete.js

```diff
@@ -21,6 +26,10 @@ module.exports = (line) => {
 
   if (split.length === 1 && lastWord.indexOf('/') === 0) { // command
     matches = command(lastWord)
+  } else if (split.length === 1 && lastWord.indexOf(':') === 0) {
```

Comment:
`split.length === 1` means the emoji has to be the only word. take that out so we can do stuff like `hi kev :blue_<tab>`